indexing-service/src/test/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisorStateTest.java

```diff
@@ -2675,13 +2759,28 @@ private static Map<String, Object> getProperties()
   }
 
   private static SeekableStreamSupervisorTuningConfig getTuningConfig()
+  {
+    return getTuningConfigWithWorkerThreads(1);
+  }
+
+  private static SeekableStreamSupervisorTuningConfig getTuningConfigWithWorkerThreads()
```

Comment:
Nit: since we are not really specifying any value for `workerThreads` in this method, it shouldn't be included in the name.

```suggestion
  private static SeekableStreamSupervisorTuningConfig createSupervisorTuningConfig()
```